simulation_parameters/common/gallery.json

```diff
@@ -0,0 +1,86 @@
+{
+    "General": {
+        "artworks": [
+            {
+                "ResourcePath": "res://assets/textures/gui/planet.png"
+            },
+            {
+                "Title": "Tribe Gang",
+                "Artist": "Uniow",
+                "ResourcePath": "res://assets/concept_arts/tribe-gang.jpg"
```

Comment:
you forgot to update the paths in this file

Reply:
Woops...